setup.py

```diff
@@ -49,16 +49,15 @@
     author_email='pipermerriam@gmail.com',
     url='https://github.com/ethereum/eth-tester',
     include_package_data=True,
-    install_requires=[
-        "toolz>0.8.2,<1;implementation_name=='pypy'",
-        "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
+    install_requires=[    
+        "eth-abi>=1.0.0,<2",
```

Comment:
Still getting this
```
eth-utils 1.2.2 has requirement eth-typing<2.0.0,>=1.3.0, but you'll have eth-typing 2.0.0 which is incompatible.
py-evm 0.2.0a33 has requirement eth-typing<2.0.0,>=1.1.0, but you'll have eth-typing 2.0.0 which is incompatible.
```
due to the eth-abi mention [here](https://github.com/ethereum/web3.py/issues/1056#issuecomment-428420981).